test/tests/functional/pbs_verify_log_output.py

```diff
@@ -113,31 +112,30 @@ def test_if_info_add(self):
         """
         Test for interface info presence in log files
         """
-        names = "["
-        for name in [x[0] for x in self.all_interfaces()]:
-            names += "(" + name + ")"
-        names += "]"
+        interfaceGenerator = self.all_interfaces()
+        for ifname in interfaceGenerator:
+            name = "[(" + ifname + ")]"
 
-        log_val = "[(ipv4)(ipv6)(ipv4/ipv6)] [(interface: )]" + names
+            log_val = "[( interface: )]" + name
```

Comment:
Having [(interface)] would mean the same thing, except PTL fails to match the string in the server logs (mom and sched logs match) for some reason. Which is why I added it inside the [] and it started working.